app/src/Components/Base/Root.js

```diff
@@ -14,7 +14,7 @@ function getCookieValue(a) {
 
 
 const client = new ApolloClient({
-  uri: process.env.REACT_APP_GRAPHQL_API_URL + 'graphql',
+  uri: process.env.REACT_APP_FLASK_API_URL + 'graphql',
```

Comment:
Is the documentation updated?

Reply:
There's `REACT_APP_FLASK_API_HOST`, `REACT_APP_GRAPHQL_API_URL` and `REACT_APP_FLASK_API_URL`, which is confusing.

Reply:
I guess, just use the one from the documentation on `development`, `REACT_APP_FLASK_API_URL`. We should just go with that one. Looks good to me.